### PR TITLE
adding the documentation for Commons APIs.

### PR DESCRIPTION
**Pull request by mdrodas**
addding commons-apis.md and addding the corresponding link in sidebars.js

### CODE REVIEW

File: docs/platform/commons-apis.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+```
+
+
+## Implementation Organization
+
+From the logical architecture of the WeNet Platform we have two related modules: Commons APIs and ILogbase.
+
+## Repository
+
+1. Commons APIs (and streambase) https://github.com/InternetOfUs/streambase
```
Comment (carlocaprini) [inline docs/platform/commons-apis.md]: @mdrodas can you please use a markdown link?

File: docs/platform/commons-apis.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+
+## Documentation
+
+1. WeNet Platform Documentation https://github.com/InternetOfUs/components-documentation
```
Comment (carlocaprini) [inline docs/platform/commons-apis.md]: @mdrodas can you please use a markdown link?

File: docs/platform/commons-apis.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
+---
+title: Commons APIs
+sidebar_label: Commons APIs
+---
+
+## What is it?
+
+Commons APIs manages streams of data that are pushed mainly from external aplications towards the WeNet Platform (example: the iLog application from the University of Trento).  This component interacts with the iLogBase component to provide the services for saving and sharing the streams to the internal component of the WeNet Infrastructure.
```
Comment (carlocaprini) [inline docs/platform/commons-apis.md]: @mdrodas can you please include details for accessing and using the application? 🙏 It is available on the app stores? If not, how is is possible to get it?

File: docs/platform/commons-apis.md [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+
+The Commons APIS provides services for the External Applications that want to collaborate with the WeNet platform by pushing data.
+
+The iLogbase provides services for handling big data within the WeNet Platform. The services include the management of streams of data with batch services and with subscription/notification services.
+
+## Implemented Streams
+
+The implemented streams are the following:
+1. Location (locationeventpertime)
```
Comment (carlocaprini) [inline docs/platform/commons-apis.md]: @mdrodas the meaning of the labels inside the parenthesis is not clear to me. Can you please make it clearer? 🙏

**Review by carlocaprini — CHANGES_REQUESTED** [review]

**Comment by mdrodas** [comment]
I added more details as requested.

**Pull request merged**